TheInternet_Pages/LoginPage.cs

```diff
@@ -0,0 +1,39 @@
+﻿using OpenQA.Selenium;
+
+namespace Authentication
+{
+    public class LoginPage
+    {
+        private readonly IWebDriver _driver;
```

Comment:
Lets make not only locators, but find these elements and do WebElement
Check everywhere

Reply:
Ok, I think I understood and done